### PR TITLE
Clean Up Find Replace Overlay Icons

### LINKED ISSUE #1977: "Replace All" icon of Search/Replace Overlay is blurry
Let's make sure issue is not already fixed in latest builds first.

- [x] I verified I can reproduce this issue against [latest Integration Build of Eclipse SDK](https://download.eclipse.org/eclipse/downloads/)

## Steps to reproduce

* Open the search replace overlay on a high resolution monitor on macOS.

I expected: 
That all icons are sharp and crisp.

But got: 
image

## Tested under this environment:
* OS & version: macOS

## Community

- [X] I understand reporting an issue to this OSS project does not mandate anyone to fix it. Other contributors may consider the issue, or not, at their own convenience. The most efficient way to get it fixed is that I fix it myself and contribute it back as a good quality patch to the project.

### PR DESCRIPTION
**Pull request by BeckerWdf**
- The icons are part of a local toolbar. Therefore move them from the "obj" folder to the "lcl16" folder.
- Fix file name for high resolution icons for select_prev and select_prev icons
- Make more clear what's the file name of and icon and what's it's key

Fixes: https://github.com/eclipse-platform/eclipse.platform.ui/issues/1977

### CODE REVIEW

**Comment by BeckerWdf** [comment]
@Wittmaxi: Can you pls. review?

**Comment by Wittmaxi** [comment]
Heiko and I debated where the icons should go. Our rationale was that the toolbar isn't semantically used as atoolbar and rather a "hack" to achieve flat-looking buttons.

I am not at all familiar with how the folders are arranged and what goes where. If you believe this is a better place for the icons (and the overlay still works!), then it probably is :)

Please remember to update the .images-repository as well!

This PR fixing a mac-issue is a true value gain! (I don't have access to a mac, so investigating into these issues is very tricky to me)

**Comment by BeckerWdf** [comment]
> I am not at all familiar with how the folders are arranged and what goes where.

See https://eclipse-platform.github.io/ui-best-practices/#folder_structure

> Please remember to update the .images-repository as well!

You you please take over that part?

**Comment by BeckerWdf** [comment]
> This PR fixing a mac-issue is a true value gain! (I don't have access to a mac, so investigating into these issues is very tricky to me)

This isn't macOS specific. What do you mean?

**Comment by Wittmaxi** [comment]
@BeckerWdf The icons didn't appear blurry on my computer, I assumed it was mac specific.

Sure, I can do that but I will do it in a few days.

**Comment by BeckerWdf** [comment]
> @BeckerWdf The icons didn't appear blurry on my computer, I assumed it was mac specific.

Then you don't have a high resolution display. This should also be visible on windows if a high res display is connected.

**Comment by BeckerWdf** [comment]
@Wittmaxi: Should we merge this?

File: bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/texteditor/FindReplaceOverlay.java [hunk 1]
```diff
@@ -629,17 +629,17 @@ private void createReplaceToggle() {
 		GridDataFactory.fillDefaults().grab(false, true).align(GridData.BEGINNING, GridData.FILL)
 				.applyTo(replaceToggle);
 		replaceToggle.setToolTipText(FindReplaceMessages.FindReplaceOverlay_replaceToggle_toolTip);
-		replaceToggle.setImage(FindReplaceOverlayImages.get(FindReplaceOverlayImages.OBJ_OPEN_REPLACE_AREA));
+		replaceToggle.setImage(FindReplaceOverlayImages.get(FindReplaceOverlayImages.KEY_OPEN_REPLACE_AREA));
```
Comment (Wittmaxi) [inline bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/texteditor/FindReplaceOverlay.java:632]:
```suggestion
		replaceToggle.setImage(FindReplaceOverlayImages.get(FindReplaceOverlayImages.OPEN_REPLACE_AREA));
```

What do you think of removing the useless prefixes? This is a perfect opportunity to test the Find/Replace overlay 😉

Comment (BeckerWdf) [reply bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/texteditor/FindReplaceOverlay.java:632]: You mean the "KEY" prefix?

Comment (Wittmaxi) [reply bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/texteditor/FindReplaceOverlay.java:632]: @BeckerWdf yes! Back when I had icons in LCL and in OBJ, the prefixes made (a bit of) sense, but now that everything is LCL, the prefix does not help in understanding the variables

Comment (BeckerWdf) [reply bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/texteditor/FindReplaceOverlay.java:632]: I added the "KEY" by intention to make it clear that this is not the filename

Comment (Wittmaxi) [reply bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/texteditor/FindReplaceOverlay.java:632]: @BeckerWdf Okay, then leave it in!

**Comment by BeckerWdf** [comment]
> Please remember to update the .images-repository as well!

I only find one commit related to the overlay there: https://github.com/eclipse-platform/eclipse.platform.images/commit/b2098e2fd9cd670032c303e4055db60b89fb44bd

All the other icons are missing there. Pls. take care of these

**Pull request merged**